client/package.json

```diff
@@ -1,11 +1,12 @@
 {
   "dependencies": {
     "@empirica/player": "^1.0.0-beta.2",
+    "@jitsi/react-sdk": "^1.0.0",
     "@react-hook/mouse-position": "4.1.3",
+    "cypress": "^9.6.1",
```

Comment:
Lets move cypress to the dev dependencies, as we won't use it on the prod server